Drive/rover_drive_system.hpp

```diff
@@ -41,20 +42,43 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
   };
 
   RoverDriveSystem(Wheel & left_wheel, Wheel & right_wheel, Wheel & back_wheel)
-      : left_wheel_(left_wheel),
-        right_wheel_(right_wheel),
-        back_wheel_(back_wheel){};
+      : left_wheel_(&left_wheel),
+        right_wheel_(&right_wheel),
+        back_wheel_(&back_wheel){};
 
   virtual void Initialize() override
   {
     sjsu::LogInfo("Initializing drive system...");
-    left_wheel_.Initialize();
-    right_wheel_.Initialize();
-    back_wheel_.Initialize();
+    left_wheel_->Initialize();
+    right_wheel_->Initialize();
+    back_wheel_->Initialize();
+    
+
     SetSpinMode();
     sjsu::LogInfo("Drive system initialized!");
   }
 
+  // example Call: drive.switchLegOrientation(drive, back_wheel, right_wheel, back_wheel)
+  void switchLegOrientation(int position)
+  {
+    /*
+    [0] = left right back
+    [1] = right back left
+    [2] = back left right
+    */
+    if(left_wheel_->GetHubSpeed() == 0 && right_wheel_->GetHubSpeed() == 0 && back_wheel_->GetHubSpeed() == 0)
+    {
+        left_wheel_= wheels[(position + 0) % 3];
+        right_wheel_ = wheels[(position + 1) % 3];
+        back_wheel_ = wheels[(position + 2) % 3];
+    }
+    else
+    {
+      sjsu::LogError("Cannot change leg orientation while wheels are moving!");
+    }
```

Comment:
This should never be ran. Our code should be smart enough to stop itself before this function ever gets called imo